### ISSUE TITLE
fix: Content and Design QA

### ISSUE THREAD

**Issue by ramboz** [hlxsites/petplace#197]
Ideally, we should all verify a random subset of pages to make sure they look good.
I'm preparing a few sets of 20 pages below for each of us to verify.

The goal is to identify remaining bugs:
- content issues during import
- layout/template issues in the CSS
- broken block behavior and UX

It is enough to document those, and we'll prioritize the fixing separately.

Summary:
- [x] https://github.com/hlxsites/petplace/issues/204
- [x] https://github.com/hlxsites/petplace/issues/205
- [ ] https://github.com/hlxsites/petplace/issues/206
- [x] https://github.com/hlxsites/petplace/issues/207
- [x] https://github.com/hlxsites/petplace/issues/208
- [x] https://github.com/hlxsites/petplace/issues/209
- [x] https://github.com/hlxsites/petplace/issues/210
- [x] https://github.com/hlxsites/petplace/issues/211
- [x] https://github.com/hlxsites/petplace/issues/212
- [x] https://github.com/hlxsites/petplace/issues/213

**Comment by ramboz** [comment]
- [ ] https://www.petplace.com/article/general/pet-behavior-training/bats-help-get-rid-of-mosquitoes/
- [ ] https://www.petplace.com/article/cats/pet-care/fading-kitten-syndrome/
- [ ] https://www.petplace.com/article/cats/pet-health/are-cats-dangerous-to-pregnant-women/
- [ ] https://www.petplace.com/article/category/pet-care/dog-care/grooming-your-dog/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/3-ways-save-pet-care-veterinarians-advice/
- [ ] https://www.petplace.com/article/general/news/new-york-banning-retail-pet-sales-puppy-mills/
- [ ] https://www.petplace.com/article/dogs/pet-care/top-dogs-with-weird-hairdos/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/are-lethargy-and-sleepiness-side-effects-of-advantage/
- [ ] https://www.petplace.com/article/drug-library/drug-library/library/triamcinolone-acetonide-vetalog-panalog-for-dogs-and-cats/
- [ ] https://www.petplace.com/article/cats/pet-care/tip-to-protect-your-cat-from-common-household-dangers/
- [ ] https://www.petplace.com/article/dogs/pet-care/great-name-ideas-for-sporting-dogs/
- [ ] https://www.petplace.com/article/horses/general/foot-and-mouth-disease-in-horses/
- [ ] https://www.petplace.com/article/dogs/pet-health/mycoplasma-in-dogs/
- [ ] https://www.petplace.com/article/dogs/just-for-fun/fall-dog-recipe/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-pets-do-the-2008-presidential-candidates-have/
- [ ] https://www.petplace.com/article/reptiles/general/caring-for-your-chameleon/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/holiday-music-cats-love/
- [ ] https://www.petplace.com/article/dogs/pet-health/eight-home-remedies-for-diy-do-it-yourself-dog-care/
- [ ] https://www.petplace.com/article/cats/pet-behavior-training/how-many-cats-are-too-many/
- [ ] https://www.petplace.com/article/general/pet-health/should-you-use-trazodone-to-manage-your-pets-anxiety-based-disorders/

**Comment by ramboz** [comment]
- [ ] https://www.petplace.com/article/dogs/pet-behavior-training/70-things-your-dog-wont-tell-you-but-wish-you-knew/
- [ ] https://www.petplace.com/article/dogs/just-for-fun/bichon-frise-wins-best-in-show-at-westminster-2001/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-are-the-anal-sacs-and-anal-glands/
- [ ] https://www.petplace.com/article/horses/general/tendon-injuries/
- [ ] https://www.petplace.com/article/dogs/just-for-fun/your-grr-eat-horoscope-for-february/
- [ ] https://www.petplace.com/article/dogs/first-aid-for-dogs/dogs-and-paper-shredder-what-i-learned-one-night-in-the-er/
- [ ] https://www.petplace.com/article/dogs/pet-health/malassezia-dermatitis-in-dogs/
- [ ] https://www.petplace.com/article/dogs/pet-health/how-to-protect-against-dog-theft/
- [ ] https://www.petplace.com/article/dogs/pet-care/how-to-clean-your-dogs-ears/
- [ ] https://www.petplace.com/article/dogs/diseases-conditions-of-dogs/trauma-to-the-chest-in-dogs/
- [ ] https://www.petplace.com/article/drug-library/drug-library/library/proparacaine-for-dogs-and-cats/
- [ ] https://www.petplace.com/article/cats/vet-qa-parent/vet-qa/why-is-my-cat-scratching-her-ear/
- [ ] https://www.petplace.com/article/reptiles/general/turtle-sales-pose-risks-to-human-health-and-animal-welfare/
- [ ] https://www.petplace.com/article/cats/pet-health/footpad-injury-in-cats/
- [ ] https://www.petplace.com/article/cats/breeds/cat-breeds/what-you-need-to-know-about-legal-exotic-cats/
- [ ] https://www.petplace.com/article/dogs/news/cloning-pets-good-idea/
- [ ] https://www.petplace.com/article/dogs/pet-health/elbow-luxation-dislocated-in-dogs/
- [ ] https://www.petplace.com/article/dogs/just-for-fun/westminster-dog-show-junior-showmanship/
- [ ] https://www.petplace.com/article/cats/pet-care/three-legged-cat-names/
- [ ] https://www.petplace.com/article/dogs/pet-behavior-training/loving-the-leash/

**Comment by ramboz** [comment]
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-is-gastric-lavage/
- [ ] https://www.petplace.com/article/cats/just-for-fun/garfield-history-facts/
- [ ] https://www.petplace.com/article/dogs/pet-care/flower-pot-scenting-game-for-dogs/
- [ ] https://www.petplace.com/article/small-mammals/general/tips-on-preventing-hairballs/
- [ ] https://www.petplace.com/article/cats/vet-qa-parent/vet-qa/why-does-my-cat-sleep-so-much/
- [ ] https://www.petplace.com/article/small-mammals/general/choosing-a-gerbil/
- [ ] https://www.petplace.com/article/cats/breeds/choosing-a-chartreux/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/why-is-my-dog-dragging-his-rear-end-on-the-floor/
- [ ] https://www.petplace.com/article/dogs/pet-behavior-training/reducing-dog-urine-odor/
- [ ] https://www.petplace.com/article/dogs/pet-health/elbow-dysplasia-in-dogs/
- [ ] https://www.petplace.com/article/cats/pet-health/fracture-of-the-digit-in-cats/
- [ ] https://www.petplace.com/article/cats/vet-qa-parent/vet-qa/urinary-obstruction-in-a-female-cat/
- [ ] https://www.petplace.com/article/dogs/pet-health/urethral-prolapse-and-neoplasia-in-dogs/
- [ ] https://www.petplace.com/article/small-mammals/general/winter-rabbit-care/
- [ ] https://www.petplace.com/article/horses/general/tips-on-feeding-hay/
- [ ] https://www.petplace.com/article/horses/general/boarding-vs-home-stabling-of-your-horse/
- [ ] https://www.petplace.com/article/cats/vet-qa-parent/vet-qa/what-can-i-do-if-my-cats-ears-freeze/
- [ ] https://www.petplace.com/article/cats/just-for-fun/lucky-cat/
- [ ] https://www.petplace.com/article/cats/vet-qa-parent/vet-qa/my-cat-is-having-some-tearing-eye-discharge-and-sneezing-what-should-i-do/
- [ ] https://www.petplace.com/article/dogs/just-for-fun/rent-a-dog-is-it-ethical/

**Comment by ramboz** [comment]
- [ ] https://www.petplace.com/article/dogs/pet-health/rhubarb-toxicity-in-dogs/
- [ ] https://www.petplace.com/article/cats/pet-care/how-to-turn-a-stray-cat-into-a-pet/
- [ ] https://www.petplace.com/article/small-mammals/general/having-fun-with-your-chinchilla/
- [ ] https://www.petplace.com/article/cats/pet-health/cat-health/cat-diet-nutrition/can-cats-be-vegan/
- [ ] https://www.petplace.com/article/dogs/pet-care/top-dog-breeds-by-city-in-2010/
- [ ] https://www.petplace.com/article/cats/pet-health/cat-health/cat-diet-nutrition/cat-obesity-growing-epidemic/
- [ ] https://www.petplace.com/article/cats/pet-health/cleft-palate-in-cats/
- [ ] https://www.petplace.com/article/small-mammals/general/low-blood-sugar-in-ferrets/
- [ ] https://www.petplace.com/article/dogs/pet-care/get-rid-of-that-winter-coat-grooming-for-your-dog-in-the-spring/
- [ ] https://www.petplace.com/article/dogs/pet-care/top-dogs-for-working-owners/
- [ ] https://www.petplace.com/article/cats/pet-health/bronchial-asthma-in-cats/
- [ ] https://www.petplace.com/article/general/pet-insurance/pet-insurance-providers-in-north-america/
- [ ] https://www.petplace.com/article/dogs/pet-health/what-you-should-do-if-your-dog-is-sprayed-with-mace-or-pepper-spray/
- [ ] https://www.petplace.com/article/dogs/pet-health/is-your-dog-at-risk-for-lyme-disease/
- [ ] https://www.petplace.com/article/dogs/pet-care/tips-for-training-your-dog-to-sit/
- [ ] https://www.petplace.com/authors/lori-savka/
- [ ] https://www.petplace.com/article/dogs/pet-health/dog-health/dog-diet-nutrition/can-dogs-eat-blueberries/
- [ ] https://www.petplace.com/article/small-mammals/general/how-to-raise-orphaned-pet-rabbits/
- [ ] https://www.petplace.com/article/dogs/pet-care/how-does-pet-insurance-differ-from-human-health-insurance/
- [ ] https://www.petplace.com/article/dogs/pet-behavior-training/dog-behavior-training/does-dog-water-intoxication-exist/

**Comment by ramboz** [comment]
- [ ] https://www.petplace.com/article/dogs/pet-care/how-to-find-good-foster-parents/
- [ ] https://www.petplace.com/article/horses/general/tips-on-parasite-control/
- [ ] https://www.petplace.com/article/cats/pet-health/the-skinny-on-cat-skin-conditions/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-is-the-history-of-the-pug/
- [ ] https://www.petplace.com/article/cats/pet-health/home-care-for-the-cat-with-diarrhea/
- [ ] https://www.petplace.com/article/horses/general/timely-deworming/
- [ ] https://www.petplace.com/give-the-gift-of-pet-insurance/
- [ ] https://www.petplace.com/article/dogs/pet-health/why-does-my-dog-need-a-dental-specialist/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/my-dog-doesnt-like-his-dry-of-wet-food-anymore/
- [ ] https://www.petplace.com/article/dogs/just-for-fun/your-grr-eat-horoscope-for-january/
- [ ] https://www.petplace.com/article/general/pet-insurance/comparing-pet-insurance-plans/
- [ ] https://www.petplace.com/article/cats/pet-care/names-for-your-fat-cat/
- [ ] https://www.petplace.com/article/birds/general/what-to-do-if-your-bird-flies-the-coop/
- [ ] https://www.petplace.com/article/dogs/breeds/dog-breeds/meet-the-vizsla/
- [ ] https://www.petplace.com/article/dogs/pet-health/how-to-prevent-the-most-common-and-expensive-dog-conditions/
- [ ] https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/head-tremors/
- [ ] https://www.petplace.com/article/horses/general/tips-on-dealing-with-stone-bruises/
- [ ] https://www.petplace.com/article/cats/diseases-conditions-of-cats/fishhook-injuries-in-cats/
- [ ] https://www.petplace.com/authors/carey-hemmelgarn-dvm-dacvecc/
- [ ] https://www.petplace.com/article/cats/pet-health/melena-blood-in-stool-in-cats/

**Comment by FentPams** [comment]
After reviewing all the pages above, there are several issues found (pls ignore it if these changes are done intentionally):
_The pages with checks ✔️have typical types of issues, but **many other pages** have the same issues._ 

1.  **Lack of Article page navigation** at the bottom (below is example):

Petplace (first pic) | Hlx.live (second pic)
-- | --
https://www.petplace.com/article/birds/general/what-to-do-if-your-bird-flies-the-coop/ | https://main--petplace--hlxsites.hlx.live/article/birds/general/what-to-do-if-your-bird-flies-the-coop

<img width="1312" alt="Screenshot 2023-06-29 at 10 19 21 PM" src="https://github.com/hlxsites/petplace/assets/105081458/3a205692-94cc-4258-a2e9-0a3f28d3eb8d">
<img width="1385" alt="Screenshot 2023-06-29 at 10 19 15 PM" src="https://github.com/hlxsites/petplace/assets/105081458/6d69a609-68cd-4161-80d3-bd013445fde2">

**Comment by FentPams** [comment]
2. Article page navigation **miss** " **> next article**" -- The article navigation doesn't show previous-thumbnail, and previous detail (display:none). But in article-navigation.css, line 1-7, they are defined. And even when the screen >1024, the next-thumbnail, next-details, next, doesn't show either.  


Petplace (1st. 2nd pic) | Hlx.live (3rd, 4th pic)
-- | --
https://www.petplace.com/article/dogs/breeds/dog-breeds/meet-the-vizsla/ | https://main--petplace--hlxsites.hlx.live/article/dogs/breeds/dog-breeds/meet-the-vizsla

-   \>1024 petplace
<img width="1272" alt="Screenshot 2023-06-29 at 10 26 26 PM" src="https://github.com/hlxsites/petplace/assets/105081458/e1511947-13e7-4adc-a1b5-b4acb36c099c">

-  <1024 petplace
<img width="421" alt="Screenshot 2023-06-29 at 10 26 34 PM" src="https://github.com/hlxsites/petplace/assets/105081458/2f87f87f-d569-4832-8eea-ff8e32749262">


-  \>1024 hlx
<img width="933" alt="Screenshot 2023-06-29 at 10 26 45 PM" src="https://github.com/hlxsites/petplace/assets/105081458/4f4fbce8-0047-4657-a3a3-aaddfab8893c">

-   <1024 hlx
<img width="251" alt="Screenshot 2023-06-29 at 10 26 39 PM" src="https://github.com/hlxsites/petplace/assets/105081458/4a75217d-044a-4c61-b6cf-5e877bda1fb7">

**Comment by FentPams** [comment]
#### 3. **h2 .subtitle** doesn't style correctly.
 In their page, many h2 has .subtitle class, with font 1.25rem. while we style it merely as h2. 


Petplace(left) | Hlx.live(right)
-- | --
https://www.petplace.com/article/cats/diseases-conditions-of-cats/fishhook-injuries-in-cats/ | https://main--petplace--hlxsites.hlx.live/article/cats/diseases-conditions-of-cats/fishhook-injuries-in-cats/
<img width="963" alt="Screenshot 2023-06-29 at 10 43 02 PM" src="https://github.com/hlxsites/petplace/assets/105081458/ab7bb88c-2f9e-4f61-953a-6b46594cca9d">

#### 4.  Markup **text issue in doc**.

 For the html structure like this: 
```
<h2>
     <p> </p>  
     <p> </p>  
          …
</h2> 
```
<img width="495" alt="Screenshot 2023-06-29 at 10 53 49 PM" src="https://github.com/hlxsites/petplace/assets/105081458/f27408f4-b919-46f5-9e88-f0f82bf484db">
<img width="210" alt="Screenshot 2023-06-29 at 11 03 55 PM" src="https://github.com/hlxsites/petplace/assets/105081458/e705af95-7077-4be4-b50a-038ac5d97213">

- https://main--petplace--hlxsites.hlx.live/article/cats/pet-health/cleft-palate-in-cats  (I have manually adjusted its doc, but I guess there may be more pages like this)


#### 5. **Blank page**

https://main--petplace--hlxsites.hlx.live/article/reptiles/general/turtle-sales-pose-risks-to-human-health-and-animal-welfare


#### 6.  "**Undefined breadcrumb**"  in authors page


Petplace | Hlx.live
-- | --
https://www.petplace.com/authors/carey-hemmelgarn-dvm-dacvecc/ | https://main--petplace--hlxsites.hlx.page/authors/carey-hemmelgarn-dvm-dacvecc

<img width="827" alt="Screenshot 2023-06-29 at 11 28 11 PM" src="https://github.com/hlxsites/petplace/assets/105081458/85aa04b5-a371-46a7-b4c6-fbda89d3a74d">

7.  **Special layouts issue**


Petplace | Hlx.live
-- | --
https://www.petplace.com/give-the-gift-of-pet-insurance/ | https://main--petplace--hlxsites.hlx.live/give-the-gift-of-pet-insurance
<img width="877" alt="Screenshot 2023-06-29 at 11 30 24 PM" src="https://github.com/hlxsites/petplace/assets/105081458/34ba50dc-9ae8-4fd9-96dd-881120bfe4a2">

8. **Theme color** mismatch 


Petplace | Hlx.live
-- | --
https://www.petplace.com/article/category/breeds/cat-breeds/ | https://main--petplace--hlxsites.hlx.live/article/cats/breeds/cat-breeds/what-you-need-to-know-about-legal-exotic-cats
<img width="870" alt="Screenshot 2023-06-29 at 11 32 31 PM" src="https://github.com/hlxsites/petplace/assets/105081458/fe94a14e-3964-45ca-9afb-24794277eee2">

9. **Text color** issue


Petplace | Hlx.live
-- | --
https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-is-the-history-of-the-pug/ | https://main--petplace--hlxsites.hlx.live/article/dogs/vet-qa-parent/vet-qa/what-is-the-history-of-the-pug
<img width="873" alt="Screenshot 2023-06-29 at 11 59 16 PM" src="https://github.com/hlxsites/petplace/assets/105081458/a07a5f72-ff24-492c-a872-080cff59115e">


10.**h3 style** in popular posts and in text
<img width="872" alt="Screenshot 2023-06-29 at 11 56 48 PM" src="https://github.com/hlxsites/petplace/assets/105081458/09fd3008-879d-4d93-8211-c858478229e2">

11. **Margin** issue
<img width="888" alt="Screenshot 2023-06-29 at 11 57 39 PM" src="https://github.com/hlxsites/petplace/assets/105081458/0b3ce858-dfe6-4c4e-8d9d-007b91fd08da">

12. **Other text style** issue:


Petplace | Hlx.live
-- | --
https://www.petplace.com/article/cats/vet-qa-parent/vet-qa/why-does-my-cat-sleep-so-much/ | https://main--petplace--hlxsites.hlx.live/article/cats/vet-qa-parent/vet-qa/why-does-my-cat-sleep-so-much
<img width="870" alt="Screenshot 2023-06-30 at 12 00 57 AM" src="https://github.com/hlxsites/petplace/assets/105081458/debdee5b-0e06-435d-8679-faa112016527">

13. **Extra symbols\*\***

Petplace | Hlx.live
-- | --
https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-is-gastric-lavage/ | https://main--petplace--hlxsites.hlx.live/article/dogs/vet-qa-parent/vet-qa/what-is-gastric-lavage
<img width="869" alt="Screenshot 2023-06-30 at 12 02 09 AM" src="https://github.com/hlxsites/petplace/assets/105081458/585045d5-2d8c-4be3-b280-7749f34c2f45">

14. **Content order** issue

Petplace | Hlx.live
-- | --
https://www.petplace.com/article/general/news/new-york-banning-retail-pet-sales-puppy-mills/ | https://main--petplace--hlxsites.hlx.live/article/general/news/new-york-banning-retail-pet-sales-puppy-mills
<img width="868" alt="Screenshot 2023-06-30 at 12 03 10 AM" src="https://github.com/hlxsites/petplace/assets/105081458/846d95a0-050a-4a4f-a803-fdd19b0ed315">

15 **Link or Button ?**

Petplace | Hlx.live
-- | --
https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-pets-do-the-2008-presidential-candidates-have/ | https://main--petplace--hlxsites.hlx.live/article/dogs/vet-qa-parent/vet-qa/what-pets-do-the-2008-presidential-candidates-have
<img width="852" alt="Screenshot 2023-06-30 at 12 03 57 AM" src="https://github.com/hlxsites/petplace/assets/105081458/bd5eba14-f479-4f03-861b-30cf2beaaeb6">


You can also refer to [my note] for details (https://adobe-my.sharepoint.com/:o:/r/personal/xfeng_adobe_com/Documents/Notebooks/[PetPlace](https://adobe-my.sharepoint.com/:o:/r/personal/xfeng_adobe_com/Documents/Notebooks/PetPlace?d=wd93813ad4f3a4df6a8d2a64701de145e&csf=1&web=1&e=KYtsir)?d=wd93813ad4f3a4df6a8d2a64701de145e&csf=1&web=1&e=KYtsir) 👈

**Comment by FentPams** [comment]
#### 3. **h2 .subtitle** doesn't style correctly.
 In their page, many h2 has .subtitle class, with font 1.25rem. while we style it merely as h2. 


Petplace(left) | Hlx.live(right)
-- | --
https://www.petplace.com/article/cats/diseases-conditions-of-cats/fishhook-injuries-in-cats/ | https://main--petplace--hlxsites.hlx.live/article/cats/diseases-conditions-of-cats/fishhook-injuries-in-cats/
<img width="963" alt="Screenshot 2023-06-29 at 10 43 02 PM" src="https://github.com/hlxsites/petplace/assets/105081458/ab7bb88c-2f9e-4f61-953a-6b46594cca9d">

**Comment by FentPams** [comment]
#### 4.  Markup **text issue in doc**.

 For the html structure like this: 
```
<h2>
     <p> </p>  
     <p> </p>  
          …
</h2> 
```
<img width="495" alt="Screenshot 2023-06-29 at 10 53 49 PM" src="https://github.com/hlxsites/petplace/assets/105081458/f27408f4-b919-46f5-9e88-f0f82bf484db">
<img width="210" alt="Screenshot 2023-06-29 at 11 03 55 PM" src="https://github.com/hlxsites/petplace/assets/105081458/e705af95-7077-4be4-b50a-038ac5d97213">

- https://main--petplace--hlxsites.hlx.live/article/cats/pet-health/cleft-palate-in-cats  (I have manually adjusted its doc, but I guess there may be more pages like this)

**Comment by FentPams** [comment]
#### 5. **Blank page**

https://main--petplace--hlxsites.hlx.live/article/reptiles/general/turtle-sales-pose-risks-to-human-health-and-animal-welfare

**Comment by FentPams** [comment]
#### 6.  "**Undefined breadcrumb**"  in authors page


Petplace | Hlx.live
-- | --
https://www.petplace.com/authors/carey-hemmelgarn-dvm-dacvecc/ | https://main--petplace--hlxsites.hlx.page/authors/carey-hemmelgarn-dvm-dacvecc

<img width="827" alt="Screenshot 2023-06-29 at 11 28 11 PM" src="https://github.com/hlxsites/petplace/assets/105081458/85aa04b5-a371-46a7-b4c6-fbda89d3a74d">

**Comment by FentPams** [comment]
#### 7.  **Special layouts issue**


Petplace | Hlx.live
-- | --
https://www.petplace.com/give-the-gift-of-pet-insurance/ | https://main--petplace--hlxsites.hlx.live/give-the-gift-of-pet-insurance
<img width="877" alt="Screenshot 2023-06-29 at 11 30 24 PM" src="https://github.com/hlxsites/petplace/assets/105081458/34ba50dc-9ae8-4fd9-96dd-881120bfe4a2">

**Comment by FentPams** [comment]
#### 8. **Theme color** mismatch 


Petplace | Hlx.live
-- | --
https://www.petplace.com/article/category/breeds/cat-breeds/ | https://main--petplace--hlxsites.hlx.live/article/cats/breeds/cat-breeds/what-you-need-to-know-about-legal-exotic-cats
<img width="870" alt="Screenshot 2023-06-29 at 11 32 31 PM" src="https://github.com/hlxsites/petplace/assets/105081458/fe94a14e-3964-45ca-9afb-24794277eee2">

**Comment by FentPams** [comment]
#### 9. **Text color** issue


Petplace | Hlx.live
-- | --
https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-is-the-history-of-the-pug/ | https://main--petplace--hlxsites.hlx.live/article/dogs/vet-qa-parent/vet-qa/what-is-the-history-of-the-pug
<img width="873" alt="Screenshot 2023-06-29 at 11 59 16 PM" src="https://github.com/hlxsites/petplace/assets/105081458/a07a5f72-ff24-492c-a872-080cff59115e">

**Comment by FentPams** [comment]
#### 10.**h3 style** in popular posts and in text
<img width="872" alt="Screenshot 2023-06-29 at 11 56 48 PM" src="https://github.com/hlxsites/petplace/assets/105081458/09fd3008-879d-4d93-8211-c858478229e2">

**Comment by FentPams** [comment]
#### 11. **Margin** issue
<img width="888" alt="Screenshot 2023-06-29 at 11 57 39 PM" src="https://github.com/hlxsites/petplace/assets/105081458/0b3ce858-dfe6-4c4e-8d9d-007b91fd08da">

**Comment by FentPams** [comment]
#### 12. **Other text style** issue:


Petplace | Hlx.live
-- | --
https://www.petplace.com/article/cats/vet-qa-parent/vet-qa/why-does-my-cat-sleep-so-much/ | https://main--petplace--hlxsites.hlx.live/article/cats/vet-qa-parent/vet-qa/why-does-my-cat-sleep-so-much
<img width="870" alt="Screenshot 2023-06-30 at 12 00 57 AM" src="https://github.com/hlxsites/petplace/assets/105081458/debdee5b-0e06-435d-8679-faa112016527">

**Comment by FentPams** [comment]
#### 13. **Extra symbols\*\***

Petplace | Hlx.live
-- | --
https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-is-gastric-lavage/ | https://main--petplace--hlxsites.hlx.live/article/dogs/vet-qa-parent/vet-qa/what-is-gastric-lavage
<img width="869" alt="Screenshot 2023-06-30 at 12 02 09 AM" src="https://github.com/hlxsites/petplace/assets/105081458/585045d5-2d8c-4be3-b280-7749f34c2f45">

**Comment by FentPams** [comment]
#### 14. **Content order** issue

Petplace | Hlx.live
-- | --
https://www.petplace.com/article/general/news/new-york-banning-retail-pet-sales-puppy-mills/ | https://main--petplace--hlxsites.hlx.live/article/general/news/new-york-banning-retail-pet-sales-puppy-mills
<img width="868" alt="Screenshot 2023-06-30 at 12 03 10 AM" src="https://github.com/hlxsites/petplace/assets/105081458/846d95a0-050a-4a4f-a803-fdd19b0ed315">

**Comment by FentPams** [comment]
#### 15 **Link or Button ?**

Petplace | Hlx.live
-- | --
https://www.petplace.com/article/dogs/vet-qa-parent/vet-qa/what-pets-do-the-2008-presidential-candidates-have/ | https://main--petplace--hlxsites.hlx.live/article/dogs/vet-qa-parent/vet-qa/what-pets-do-the-2008-presidential-candidates-have
<img width="852" alt="Screenshot 2023-06-30 at 12 03 57 AM" src="https://github.com/hlxsites/petplace/assets/105081458/bd5eba14-f479-4f03-861b-30cf2beaaeb6">

**Comment by travrob1** [comment]
Disclosure coming after table of contents
Theirs: 

![Image](https://github.com/hlxsites/petplace/assets/4155964/08109eda-8096-4302-8671-6f588f78702b)

Ours:

![Image](https://github.com/hlxsites/petplace/assets/4155964/3b6d253b-ad8d-446a-a920-c00932348f6f)

Source: https://main--petplace--hlxsites.hlx.live/article/dogs/pet-health/dog-health/dog-diet-nutrition/can-dogs-eat-blueberries#h2

**Comment by FentPams** [comment]
#### 16 The **Email us** link broken and not redirects to our 404 page or the main page.
https://main--petplace--hlxsites.hlx.live/article/dogs/pet-care/great-name-ideas-for-sporting-dogs
<img width="388" alt="Screenshot 2023-06-30 at 12 53 31 PM" src="https://github.com/hlxsites/petplace/assets/105081458/a17b9218-ecf0-4268-afbd-db6ee77aac06">
<img width="484" alt="Screenshot 2023-06-30 at 12 53 38 PM" src="https://github.com/hlxsites/petplace/assets/105081458/54666d98-bb53-43cc-b421-ae20d707061d">

**Comment by ramboz** [comment]
Since, it's a bit hard to directly comment on each, I'll use the following emoji notation to triage/track each item:

- 🚀 issue fixed and shipped to prod
- 👍 bug acknowledged and has a matching issue created in the backlog
- 👎 bug rejected, and explanation is provided in this thread

**Comment by ramboz** [comment]
For https://github.com/hlxsites/petplace/issues/197#issuecomment-1614811976, waiting on Matt to review the special pages and see what he wants to do with those. Likely post-golive

**Comment by ramboz** [comment]
For https://github.com/hlxsites/petplace/issues/197#issuecomment-1615134417, the link is also broken on their page, so we won't do anything about it. I'm reporting all the broken links separately and Matt and I will discuss how to address those in a different thread
